test/StatementInsurer/change_column_null_test.rb

```diff
@@ -126,8 +126,8 @@ def skip_if_unmet_requirements!
     end
 
     def met_requirements?
-      Gem::Requirement.new('>6.1', '<7.1').satisfied_by?(Gem::Version.new(::ActiveRecord::VERSION::STRING)) &&
-        SafePgMigrations.pg_version_num >= 120_000
+      Gem::Requirement.new('>6.1').satisfied_by?(Gem::Version.new(::ActiveRecord::VERSION::STRING)) &&
+        SafePgMigrations.get_pg_version_num(ActiveRecord::Base.connection) >= 120_000
```

Comment:
I'm not sure what should happen with 6.1 version, but maybe we could just keep:
```suggestion
      SafePgMigrations.get_pg_version_num(ActiveRecord::Base.connection) >= 120_000
```

Reply:
why do you need to remove '<7.1' ?

Reply:
6.1, not 7.1 :)